.travis.yml

```diff
@@ -11,6 +15,14 @@ branches:
 
 install:
   - npm ci
+  - git clone https://github.com/DefinitelyTyped/DefinitelyTyped ../DefinitelyTyped
+
+script:
+  - npm tes
+  - npm run build
+  - npm run clean
+  - npm run parse
+  - npm run check
 
```

Comment:
probably should `npm run calculate-versions` and `npm run generate` and `npm run index`.

Note that some of these tasks hit npm (parse, index and maybe generate?)